.github/workflows/update-renovate-prs.yml

```diff
@@ -0,0 +1,46 @@
+name: Update renovate PRs
+
+on:
+  push:
+    branches:
+      - 'renovate/**'
+
+jobs:
+  check-metadata:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+      id-token: write
+
+    env:
+      GO111MODULE: on
+    steps:
+      - name: Checkout
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3
+        with:
+          ref: ${{ github.head_ref }}
+
+      - uses: actions/setup-python@bd6b4b6205c4dbad673328db7b31b7fab9e241c0 # v4
+        with:
+          python-version: 3.7
+
+      - uses: actions/setup-go@fac708d6674e30b6ba41289acaab6d4b75aa0753 # v4
+        with:
+          go-version: ^1
```

Comment:
NP: Should we set a specific Go version here? Or will it just always default to whatever is available for the latest?

Reply:
Yes, we don't care about golang version.

Reply:
Great